lib/template/config/puma.rb

```diff
@@ -1,6 +1,6 @@
 require "./config/config"
 
-environment Config.rack_env
+environment Config.pliny_env
```

Comment:
And also: maybe just remove `rack_env` from the configuration template. It doesn't seem like we actually depend on it anywhere anymore.